app/entities/user.go

```diff
@@ -13,18 +13,25 @@ type User struct {
 	LastName  string `json:"last_name"`
 	Password  string `json:"password"`
 	CreatedAt string `json:"created_at"`
+	DeletedAt string `json:"deleted_at"`
```

Comment:
I propose rename field as `ModifiedA` and `json:"modified_at"`, because in fact the user is not even deleted. And another reason the field `ModofiedAt` will be needed soon. So after the user was marked as deleted, the field `ModofiedAt` will point on the  date of the action, because nothing will happen to the user after that